percy/providers/app_automate.py

```diff
@@ -30,6 +31,22 @@ def set_debug_url(self, session_details):
         session_hash = session_details.get("sessionHash")
         self.debug_url = "https://app-automate.browserstack.com/dashboard/v2/builds/" + build_hash + "/sessions/" + session_hash
 
+    def _get_tiles(self, **kwargs):
+        fullpage_ss = kwargs.get('fullpage_screenshot', False)
+        if fullpage_ss:
+            tile_count = kwargs.get('num_of_tiles', 4)
```

Comment:
This param name seems like is not decided yet. I see different in all SDKs. lets decide on this in standup and update all sdks accordingly